src/use-image.hook.js

```diff
@@ -3,9 +3,14 @@ import {useDocumentExecCommand} from './use-document-exec-command.hook.js'
 import {RichTextContext} from './rich-text-container.component.js'
 
 const noop = () => {}
-const defaultOpts = {processImgElement: noop, fileBlobToUrl: defaultFileBlobToUrl}
+const defaultAcceptImgTypes = '.jpg, .png, image/*'
+const defaultOpts = {
+  processImgElement: noop,
+  fileBlobToUrl: defaultFileBlobToUrl,
+  acceptImgTypes: defaultAcceptImgTypes,
+}
```

Comment:
since `defaultOpts` was getting fairly long, just split it into lines and added the `acceptImgTypes` option.